includes/polling/wifi.inc.php

```diff
@@ -7,6 +7,8 @@
     } elseif ($device['os'] == 'airos-af') {
         echo 'It Is AirFIBER' . PHP_EOL;
         include 'includes/polling/mib/ubnt-airfiber-mib.inc.php';
+        echo 'It Is AirFIBER - Rx' . PHP_EOL;
+        include 'includes/polling/wireless/ubnt-airfibre-rx.inc.php';
```

Comment:
It could be better to just modify poll_mib_def() to accept an rrd_name to be passed into $tags so you can run it more than once potentially.

If that's not possible then I still think the rx code should just go in the includes/polling/mib/ubnt-airfiber-mib.inc.php file.